README.md

```diff
@@ -85,6 +85,12 @@ $ docker-compose pull
 $ docker run --rm -ti --network=host vesoft/nebula-console:nightly --addr=127.0.0.1 --port=3699
 ```
 
+If the authorization is not enabled, **Nebula Graph** will use the default user name and password to log in, so you can skip the user and password specification here. If the authorization is enabled, the user and password must be specified. For example:
```

Comment:
I think it's better to describe how to enable authorization, like configure `--enable_authorize=true` in graphd.conf